src/std/fs.ab

```diff
@@ -1,4 +1,4 @@
-import { join, len, replace_regex, split } from "std/text"
+import { join, replace_regex, split } from "std/text"
```

Comment:
We have `if len(paths) == 1` in the definition of `glob_multiple()` below. Is this going to break, now `len` is a builtin? I know you've suggested we might want to make the parentheses syntax mandatory, but do we already handle it as an alternative?

Reply:
We do already handle this alternative. It's the same as `not(true)` or in other words `not ( true )` basically it's an unary operator applied to an expression that is wrapped around parentheses